include/SFML/Graphics/Text.hpp

```diff
@@ -299,14 +375,45 @@ class SFML_GRAPHICS_API Text : public Drawable, public Transformable
     void ensureGeometryUpdate() const;
 
     ////////////////////////////////////////////////////////////
+    /// \brief Adds a line to the vertices
+    ///
+    /// This used for adding strikethrough and underline lines
+    ///
+    /// \param position         Width and top of the line
```

Comment:
Sorry for the delay (had to re-familiarize myself with the code)... the reason it was like this is because in the ```Text::ensureGeometryUpdate()``` function the current character position is tracked as ```x``` and ```y``` variables. It also happened to be that ```x``` was also the width of current text line when adding an outline/underline line (need more "line"s?) and ```y``` was the top.

So when I wrote the ```addLine(...)``` function I grouped the two variables (now as width and top) together in a vector. Looking over the code it seems now more reasonable to split that vector and name the variables ```lineLength``` and ```lineTop```. The PR now reflects these changes.